corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -146,6 +146,18 @@ def commcare_fixture_instances(domain, instance_name):
         return Instance(id=instance_name, src='jr://fixture/{}'.format(instance_name))
 
 
+@register_factory('commcare-reports')
+def commcare_reports_fixture_instances(domain, instance_name):
+    if instance_name.startswith('commcare-reports:') and toggles.MOBILE_UCR.enabled(domain):
```

Comment:
Why do we check the toggle here? Doesn't this only get called if the instance name is referenced in the app? If that's the case won't this break the app by not returning the instance?

I mean the app won't really work anyway since the fixture won't exists if the toggle isn't enabled but just thinking that might be an easier to debug failure mode.

Reply:
Good question. I think it could possibly be removed. I originally just made it hte same as https://github.com/dimagi/commcare-hq/blob/314fe154ff8bf1e2f15184f5911054e45e89ad17/corehq/apps/app_manager/suite_xml/post_process/instances.py#L144